lib/safe-pg-migrations/base.rb

```diff
@@ -53,10 +53,15 @@ def close_alternate_connection
       @alternate_connection = nil
     end
 
-    ruby2_keywords def say(*args)
+    def say(message, subitem = false, sensitive: false)
       return unless current_migration
 
-      current_migration.say(*args)
+      sensitive_logger = SafePgMigrations.config.sensitive_logger
+
+      return current_migration.say message, subitem unless sensitive_logger && sensitive
+
+      current_migration.say 'Sensitive data sent to sensitive logger', subitem
+      sensitive_logger.info(message)
```

Comment:
Should we add some context here to help?

Reply:
Sure! What would you like to see?

Reply:
Something to identify which migration is in progress.

Reply:
Here it is: https://github.com/doctolib/safe-pg-migrations/pull/101/files#diff-d83192e6fcad30c8c78dcd58221003962d2b35a3458ef9785766196aa9f95acbR26-R27